uitools/toolkitcpp/cpp/Esri/ArcGISRuntime/Toolkit/BarChartPopupMediaItem.cpp

```diff
@@ -0,0 +1,130 @@
+/*******************************************************************************
+ *  Copyright 2012-2025 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+#include "BarChartPopupMediaItem.h"
+
+// Qt headers
+#include <QBarSet>
+#include <QColor>
+
+// Maps SDK headers
+#include <PopupMedia.h>
+#include <PopupMediaValue.h>
+
+// Toolkit headers
+#include <PopupMediaItem.h>
+
+namespace Esri::ArcGISRuntime::Toolkit {
+
+/*!
+  \internal
+  This class is an internal implementation detail and is subject to change.
```

Comment:
Now that these are added, I would be very clear and remove everything else. I think `\internal` takes precedence in the doc build itself, but a user might overlook that and see the doc that's still here like the `\brief` below and assume this is fair game to use directly in their code.

Reply:
Done.